test/jdk/java/util/Locale/LocaleProvidersRun.java

```diff
@@ -172,6 +172,9 @@ public static void main(String[] args) throws Throwable {
 
         //testing 8248695 fix.
         testRun("HOST", "bug8248695Test", "", "", "");
+
+        //testing 8257964 fix. (macOS/Windows only)
+        testRun("HOST", "bug8257964Test", "", "", "");
```

Comment:
This test runs only if the platform locale is en-GB. Do we know if the test system run tests on multiple locales? From the console output unfortunately, it's impossible to tell which specific tests were run

Reply:
Could be, but it is enough to test one locale to demonstrate platform settings are used for minimal days in first week. Each test case name can be found in the command line to the java launcher which is logged in the .jtr file.

Reply:
Good to know it's shown in the .jtr file. Let's hope a link to the test result (html) will be provided in the future :-)